packages/graphql-validate-fixtures/test/fixtures/ambiguous-operation/fixtures/projectA/another-another-fixture.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I know the boilerplate is awful for these tests, but there's too much happening in this fixture.  I have to read the snapshot file to be sure that `another-fixture` and `another-another-fixture` independently cause errors.

Reply:
these multi-project fixtures are just complex by the nature of what they are testing. i can simplify them down a little so that they only test one type of ambiguous detection, but that would multiple the added files even further. For this particular fixture, the min required content is `graphqconfig`, two schemas, two queries, and two fixtures. The extra files are just testing different methods of detecting the ambiguity. That is perhaps not necessary given we have other tests that cover how fixtures map to operation names.

Reply:
gave this some thought overnight, I will remove the additional tests in these fixtures because they are really covered in a different test. That should reduce their complexity down quite a bit.